generator/ServiceModels/_sdk-versions.json

```diff
@@ -6,3091 +6,3099 @@
     "OverrideCoreVersion" : "4.0",
     "DefaultToPreview"    : true,
     "PreviewLabel"        : "preview.2",
-    "ServiceVersions"     : {
```

Comment:
I would not change the indentation of this file (it's hard to see what changed without your comment and will make the next V3 -> V4 sync harder).

What about adding the package at the end (similar to what the build system already does for new packages):
```json
        "SSMQuickSetup"                               : {
            "Version" : "4.0.0.0",
            "AssemblyVersionOverride" : "4.0",
            "Dependencies"            : {
                "Core" : "4.0.0.0"
            },
            "InPreview"               : false
        },
        "DynamoDBStreams"                       : {
            "Version" : "4.0.0.0",
            "AssemblyVersionOverride" : "4.0",
            "Dependencies"            : {
                "Core" : "4.0.0.0"
            },
            "InPreview"               : false
        }
    },
    "DefaultAssemblyVersion" : "4.0"
```

Reply:
fixed! VS auto-formatted it which made it look weird. and added to the end